src/renderer/shared/ui-kit/Select/Select.tsx

```diff
@@ -85,8 +85,8 @@ const Button = ({ placeholder }: TriggerProps) => {
         },
       )}
     >
-      <div className="overflow-hidden">
-        <RadixSelect.Value placeholder={placeholder} />
+      <div className="flex-1 overflow-hidden text-left">
```

Comment:
For some reason `Select` centers content if it occupies all width, but in our components we never intent to do so

Reply:
It's better to use `text-start`